websocket/src/desktop.rs

```diff
@@ -245,6 +246,38 @@ where
     }
 }
 
+fn client_addr_to_ws(client_addr: &Multiaddr, is_wss: bool) -> String {
+    let inner = {
+        let protocols: Vec<_> = client_addr.iter().collect();
+
+        if protocols.len() != 2 {
+            "127.0.0.1".to_owned()
```

Comment:
Shouldn't we error in case the address does not conform to the required format?

Reply:
Whatever this function returns, what we do in reality is pass the inner multiaddress to the underlying transport, then negotiate websockets over the socket provided by the underlying transport. Therefore the actual IP address here doesn't matter.